logic.py

```diff
@@ -46,3 +66,18 @@ def wordcreator(word,partofspeech): # this function takes word selected and part
         return word
     else:
         print("Not Found")
+
+def SQL_log(statment,status,purpose):
```

Comment:
A small description of this function would be great to make things more understandable.

Reply:
done